public_html/templates/footer.html

```diff
@@ -1,39 +1,45 @@
 <footer>
-            <div class="footer" style="margin-top:100px">
-            <p>Copyright  2016 - TNS. All Rights Reserved</p>
+            <div class="footer" style="margin-top:75px">
                 <div class="row">
                     <div class="third">
-                        <h3 class="footer-header">Customer Service</h3>
-                        <div><a href="comingsoon.html">Frequently Asked Questions</a></div>
-                        <div><a href="comingsoon.html">Returns</a></div>
+                        <h3 class="footer-header">Site Map</h3>
+                        <div><a href="comingsoon.html">Women</a></div>
+                        <div><a href="comingsoon.html">Men</a></div>
                         <div><a href="comingsoon.html">Live Chat</a></div>
                         <div><a href="comingsoon.html">Track Orders</a></div>
-                        <div><a href="contact.html">Contact Us</a></div>
                     </div>
                     <div class="third">
                         <h3 class="footer-header">Company Info</h3>
                         <div><a href="about.html">About Us</a></div>
-                        <div><a href="comingsoon.html">Complain about Us</a></div>
+                        <div><a href="contact.html">Contact Us</a></div>
+						<br>
+						<form>
+							Join our mailing list!<br>
+							<input type="text" name="donothing">
+							<input type="submit" value="Subscribe"
```

Comment:
should be closed off

`<input type .... name="do-nothing" />
<input type..... value="subscribe" />`